etc/config/hook.defaults.properties

```diff
@@ -0,0 +1,12 @@
+# Default settings for Hook
+
+supportsBinary=false
+interpreted=true
+supportsExecute=false
+versionFlag=--version
+compilerType=hook
+needsMulti=false
+compilers=hook
+defaultCompiler=hook
+compiler.hook.exe=hook
+compiler.hook.name=Hook
```

Comment:
Also, for the `.amazon` file, could you please fill the license info for those compilers?
 As of a few minutes ago, we're asking compiler groups to provide license info
`group.hook.licensePreamble` for custom text to be shown,
`group.hook.licenseName` for the license name,
`group.hook.licenseLink` for either a link to the license or other relevant URL.

All three are optional if you don't have anything for the preamble, for example

Reply:
Hook uses MIT License. Is it enought to put the following? 
```
group.hook.licenseName=MIT
group.hook.licenseLink=http://opensource.org/licenses/MIT
```

Reply:
We're not yet set as to what to set in the link, but maybe https://github.com/fabiosvm/hook-lang/blob/main/LICENSE works better

Reply:
Good.